pdseg/loss.py

```diff
@@ -77,7 +77,7 @@ def softmax_with_loss(logit,
         weighted_label_one_hot.stop_gradient = True
 
     loss = loss * ignore_mask
-    avg_loss = fluid.layers.mean(loss) / fluid.layers.mean(ignore_mask)
+    avg_loss = fluid.layers.mean(loss) / (fluid.layers.mean(ignore_mask) + 0.00001)
```

Comment:
Use a global epsilon variable to store it, don't use hard code.

Reply:
Done. Thanks!

Reply:
Done. Thanks!